synapseclient/client.py

```diff
@@ -2080,10 +2266,10 @@ def _getRawAnnotations(self, entity, version=None):
 
     @deprecated.sphinx.deprecated(
         version="2.1.0",
-        reason="deprecated and replaced with :py:meth:`get_annotations`",
+        reason="deprecated and replaced with get_annotations][synapseclient.Synapse.get_annotations]",
```

Comment:
This is going to conflict with the changes I have in:
https://github.com/Sage-Bionetworks/synapsePythonClient/pull/1041/files#diff-848a7a4238d8f0a7792848097e227f8e3aa284aeadf15e1fa3d890f17791b89cR1996

Also: These decorators do not make it to the mkdocs site, so having the `[text][reference.to.code]` does not add context. This is the message displayed in the console after this method is called. In these decorators we should plan to just use the new name of the method as in the referenced code.

Reply:
Fixed.